foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/CountAction.java

```diff
@@ -105,4 +104,15 @@ public ActionResponse execute(CountRequest parameter) throws QueryStoreException
                     "Error running count query.", e);
         }
     }
+
+    @Override
+    protected boolean parameterTableExists(CountRequest parameter) {
```

Comment:
Instead of putting validation for tables only, create a generic method for input validation. Also, no need to pass parameter (Use getParameter()) directly